server/app.js

```diff
@@ -17,6 +21,15 @@ import {
 const staticDir = path.join(__dirname, "static");
 
 const app = express();
+app.use(session({
+	secret: "secretcode",
```

Comment:
This is a property that we want to be an environment variable in the `.env` file

Reply:
sure I will change it